src/com/example/homework/homeworkFive/TestStudent.java

```diff
@@ -0,0 +1,23 @@
+package com.example.homework.homeworkFive;
+
+public class TestStudent {
```

Comment:
Please create the main method in this class  and create objects in the main method such way:
Student student1 = new Student();  - Please do it with one line of code